### DISCUSSION TITLE
404 only when stopping time

### DISCUSSION THREAD

**Discussion by Solidus-Prime** [kimai/kimai#2286] [Questions & Answers]
Hi,

We are getting a 404 error only when an employee tries to stop their time. Everything else seems to work perfectly, including starting a timesheet.

The webserver does direct to the public folder as noted in the installation instructions.

**Comment by kevinpapst** [comment]
If that error occurs in the demo: it is a bug.
If that error occurs only in your installation: it is an environment issue.
Check the logs, check your browser console and post more information. With that little info it is impossible to help.

**Comment by Solidus-Prime** [comment]
This is from var/logs

 request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\MethodNotAllowedHttpException: "No route found for "GET /api/timesheets/9/stop": Method Not Allowed (Allow: PATCH)" at C:\inetpub\wwwroot\kimai2\vendor\symfony\http-kernel\EventListener\RouterListener.php line 140 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\MethodNotAllowedHttpException(code: 0): No route found for \"GET /api/timesheets/9/stop\": Method Not Allowed (Allow: PATCH) at C:\\inetpub\\wwwroot\\kimai2\\vendor\\symfony\\http-kernel\\EventListener\\RouterListener.php:140, Symfony\\Component\\Routing\\Exception\\MethodNotAllowedException(code: 0):  at C:\\inetpub\\wwwroot\\kimai2\\vendor\\symfony\\routing\\Matcher\\Dumper\\CompiledUrlMatcherTrait.php:42)"} []

**Reply by kevinpapst** [reply]
Check your browser console if the request is really sent as GET. I assume it is correctly sent aas PATCH and you are using a reverse proxy or webserver configs, that disallows to use the HTTP method PATCH.
